# libcss: build
# 2 files, 16 changed lines
#
# Regenerate: tools/regen-patches.sh && tools/split-patches.py
#
diff -urN -x .git -x 'build-*' -x test -x '*.o' -x '*.a' -x '*.d' -x '*.pc' upstream/libcss/src/parse/properties/Makefile browser/libcss/src/parse/properties/Makefile
--- upstream/libcss/src/parse/properties/Makefile	2026-09-20 23:27:15.834693962 +0100
+++ browser/libcss/src/parse/properties/Makefile	2026-09-20 23:19:17.183665876 +0100
@@ -35,9 +35,11 @@
 	background_position.c		\
 	border.c			\
 	border_color.c			\
+	p_border_radius.c		\
 	border_spacing.c		\
 	border_style.c			\
 	border_width.c			\
+	p_box_shadow.c			\
 	clip.c				\
 	columns.c			\
 	column_rule.c			\
@@ -46,6 +48,7 @@
 	cursor.c			\
 	elevation.c			\
 	fill_opacity.c			\
+	p_fill.c			\
 	flex.c				\
 	flex_flow.c			\
 	font.c				\
@@ -59,7 +62,12 @@
 	overflow.c			\
 	padding.c			\
 	pause.c				\
+	p_macsurf_grid_flow.c		\
+	p_macsurf_hstripe_bg.c		\
+	p_macsurf_dotgrid.c		\
+	p_macsurf_object_position.c	\
 	play_during.c			\
+	p_pointer_events.c		\
 	properties.c			\
 	quotes.c			\
 	stroke_opacity.c		\
diff -urN -x .git -x 'build-*' -x test -x '*.o' -x '*.a' -x '*.d' -x '*.pc' upstream/libcss/src/select/properties/Makefile browser/libcss/src/select/properties/Makefile
--- upstream/libcss/src/select/properties/Makefile	2026-09-20 23:27:15.837056084 +0100
+++ browser/libcss/src/select/properties/Makefile	2026-09-20 23:19:17.228821379 +0100
@@ -16,6 +16,7 @@
 border_left_color.c \
 border_left_style.c \
 border_left_width.c \
+s_border_radius.c \
 border_right_color.c \
 border_right_style.c \
 border_right_width.c \
@@ -24,6 +25,7 @@
 border_top_style.c \
 border_top_width.c \
 bottom.c \
+s_box_shadow.c \
 box_sizing.c \
 break_after.c \
 break_before.c \
@@ -51,6 +53,7 @@
 elevation.c \
 empty_cells.c \
 fill_opacity.c \
+s_fill.c \
 flex_basis.c \
 flex_direction.c \
 flex_grow.c \
@@ -86,6 +89,10 @@
 outline_width.c \
 overflow_x.c \
 overflow_y.c \
+s_macsurf_grid_flow.c \
+s_macsurf_hstripe_bg.c \
+s_macsurf_dotgrid.c \
+s_macsurf_object_position.c \
 padding_bottom.c \
 padding_left.c \
 padding_right.c \
@@ -98,6 +105,7 @@
 pitch.c \
 pitch_range.c \
 play_during.c \
+s_pointer_events.c \
 position.c \
 quotes.c \
 richness.c \
